frinx/common/util.py

```diff
@@ -0,0 +1,45 @@
+import json
+from typing import Any
+
+import requests
+
+
+def jsonify_description(
+    description: str, labels: list[str] | None = None, rbac: list[str] | None = None
+) -> str:
+    """Returns description in format of stringified JSON.
+    >>> jsonify_description("Hello world")
+    '{"description": "Hello world"}'
+    >>> jsonify_description("Hello world", labels=["A", "B"])
+    '{"description": "Hello world", "labels": ["A", "B"]}'
+    >>> jsonify_description("Hello world", labels=["A", "B"], rbac=["C", "D"])
+    '{"description": "Hello world", "labels": ["A", "B"], "rbac": ["C", "D"]}'
+    """
+    desc_representation: dict[str, Any] = {'description': description}
+    if labels:
+        desc_representation['labels'] = labels
+    if rbac:
+        desc_representation['rbac'] = rbac
+    output = json.dumps(desc_representation)
+    return output
+
+
+def parse_response(response: requests.Response) -> tuple[int, str]:
```

Comment:
This function is not used anywhere in the code.

Reply:
removed from codebase